packages/SketchMorph2-Core.package/M2DrawingComponent.class/properties.json

```
@@ -0,0 +1,19 @@
+{
+	"category" : "SketchMorph2-Core",
+	"classinstvars" : [
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "",
+	"instvars" : [
+		"context",
+		"form",
+		"eraser",
```

Instance variables should be ordered alphabetically.
```suggestion
		"eraser",
		"form",
```